README.md

```diff
@@ -93,15 +93,13 @@ When you register a new account you are asked for an email address to send valid
 * Support for client-side data caching.
 * Ability to block unwanted communication server-side.
 * Anonymous users (important for use cases related to tech support over chat).
-* Push notifications using [FCM](https://firebase.google.com/docs/cloud-messaging/) or [TNPG](server/push/tnpg/).
```

Comment:
Why remove this?

Reply:
I want to cull the list of features by removing expected/trivial stuff. The current list is just too long and makes it hard to see important features.

Reply:
I've trimmed it down a bit more + added missing features.